pkg/plugins/platform/local.go

```diff
@@ -242,22 +245,42 @@ func (l *LocalApp) ImportDB(imPath string, extPath string) error {
 	return nil
 }
 
-// SiteStatus returns the current status of an application based on the web container.
+// SiteStatus returns the current status of an application determined from web and db service health.
 func (l *LocalApp) SiteStatus() string {
-	webContainer, err := l.FindContainerByType("web")
-	if err != nil {
-		return SiteNotFound
-	}
+	var siteStatus string
+	services := map[string]string{"web": "", "db": ""}
 
-	status := dockerutil.GetContainerHealth(webContainer)
-	if status == "exited" {
-		return SiteStopped
+	for service := range services {
+		container, err := l.FindContainerByType(service)
+		if err != nil {
+			services[service] = SiteNotFound
+			siteStatus = service + " service " + SiteNotFound
+		} else {
+			status := dockerutil.GetContainerHealth(container)
+
+			switch status {
+			case "exited":
+				services[service] = SiteStopped
+				siteStatus = service + " service " + SiteStopped
+			case "healthy":
+				services[service] = SiteRunning
+			default:
```

Comment:
Does restarting need its own entry or is default enough? We've seen restarting more often than any other problem case I think.